book/setup_mac.md

```diff
@@ -863,23 +863,23 @@ PhpStorm can automatically look at your code and warn you of lines that do not m
 Use the following settings:
 
 - Configuration: `System PHP`
-- Check files with extensions: `php,js,css,inc, module`
+- Check files with extensions: `php, js, css, inc, module`
```

Comment:
I don't think we want spaces here but I don't really know if this will goof up PHPStorm or not - we should probably remove them just in case.